engine/src/main/java/org/terasology/engine/modes/StateMainMenu.java

```diff
@@ -43,134 +44,140 @@
 import org.terasology.utilities.Assets;
 
 /**
- * The class implements the main game menu.
- * <br><br>
+ * The class implements the main game menu. <br>
+ * <br>
  *
  * @version 0.3
  */
 public class StateMainMenu implements GameState {
-    private Context context;
-    private EngineEntityManager entityManager;
-    private EventSystem eventSystem;
-    private ComponentSystemManager componentSystemManager;
-    private NUIManager nuiManager;
-    private InputSystem inputSystem;
+	private Context									context;
```

Comment:
Yeah I think that was from switching from my default Eclipse environment to a IntelliJ one and not having styles set up, but will definitely reformat this PR.